README.md

```diff
@@ -322,11 +320,7 @@ class ApplicationController < ActionController::Base
 
   def stoplight(&block)
     Stoplight("#{params[:controller]}##{params[:action]}")
-      .with_fallback do |error|
-        Rails.logger.error(error)
-        render(nothing: true, status: :service_unavailable)
-      end
-      .run(&block)
+      .run(-> { render(nothing: true, status: :service_unavailable) }, &block)
```

Comment:
Should we keep the fallback (but with the new implementation) in this example?

Reply:
Honestly, I don't think calling logger from fallback function is an intedended fallback use considering Stoplight offers configurable errors notifiers. Another issues with the original example is that when Stoplight is red, it will pass `nil` to the follback function which leads to calling `Ralis.logger.error(nil)`. This is not a new beheviour, it was implemented this way before. 

So I wanted to update this example to showcase a more ideomatic use case. Does it make sense?